datasets/librispeech.py

```diff
@@ -68,20 +67,20 @@ def run(self):
                     md5_out.write(line)
                     break
 
-        checksum_result = subprocess.check_output(
-            ["md5sum", "-c", "md5sum-%s.txt" % self.corpus_key]
-        )  #  type: bytes
-        assert b"OK" in checksum_result
-
+        subprocess.check_call(
+            ["md5sum", "--status", "-c", "md5sum-%s.txt" % self.corpus_key]
+        )
         subprocess.check_call(
             [
                 "tar",
                 "-xf",
                 "%s.tar.gz" % self.corpus_key,
                 "-C",
-                self._out_parent_folder.get_path(),
+                ".",
             ]
         )
+        shutil.move("LibriSpeech", self._out_archive_folder.get_path())
```

Comment:
```suggestion
        shutil.move(os.path.join("LibriSpeech", self.corpus_key), self.out_corpus_folder.get_path())
```

Reply:
This will not work, because there is also content in the "LibriSpeech" folder itself.

Reply:
I'd suggest to update the DownloadMetadataJob and make it more specific.